cmd/orb_test.go

```diff
@@ -2158,7 +2161,104 @@ query namespaceOrbs ($namespace: String, $after: String!) {
 			})
+		})
+
+		Describe("when listing private orbs", func() {
+			BeforeEach(func() {
+				By("setting up a mock server")
+
+				query := `
+query namespaceOrbs ($namespace: String, $after: String!, $view: OrbListViewType) {
+	registryNamespace(name: $namespace) {
+		name
+                id
+		orbs(first: 20, after: $after, view: $view) {
+			edges {
+				cursor
+				node {
+					versions {
+						source
+						version
+					}
+					name
+	                                statistics {
+		                           last30DaysBuildCount,
+		                           last30DaysProjectCount,
+		                           last30DaysOrganizationCount
+	                               }
+				}
+			}
+			totalCount
+			pageInfo {
+				hasNextPage
+			}
+		}
+	}
+}
+`
+
+				request := graphql.NewRequest(query)
+				request.Variables["after"] = ""
+				request.Variables["namespace"] = "circleci"
+				request.Variables["view"] = "PRIVATE_ONLY"
+
+				encodedRequest, err := request.Encode()
+				Expect(err).ShouldNot(HaveOccurred())
+
+				tmpBytes := golden.Get(GinkgoT(), filepath.FromSlash("gql_orb_list_with_namespace/second_response.json"))
+				mockResponse := string(tmpBytes)
 
+				tempSettings.AppendPostHandler("", clitest.MockRequestResponse{
+					Status:   http.StatusOK,
+					Request:  encodedRequest.String(),
+					Response: mockResponse,
+				})
+			})
+
+			It("returns an error when private is provided without a namespace", func() {
+				command = exec.Command(pathCLI,
+					"orb", "list",
+					"--private",
+					"--skip-update-check",
+					"--host", tempSettings.TestServer.URL(),
+				)
+
+				By("running the command")
+				session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+
+				Expect(err).ShouldNot(HaveOccurred())
+				Eventually(session.Err).Should(gbytes.Say("Namespace must be provided when listing private orbs"))
+				Eventually(session).Should(clitest.ShouldFail())
+				Expect(tempSettings.TestServer.ReceivedRequests()).Should(HaveLen(0))
+			})
+			It("successfully returns private orbs within a given namespace", func() {
+				command = exec.Command(pathCLI,
+					"orb", "list", "circleci",
+					"--private",
+					"--skip-update-check",
+					"--host", tempSettings.TestServer.URL(),
+				)
+
+				By("running the command")
+				session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+
+				Expect(err).ShouldNot(HaveOccurred())
+				stdout := session.Wait().Out.Contents()
+				Expect(string(stdout)).To(Equal(`Orbs found: 5. Showing only certified orbs.
+Add --uncertified for a list of all orbs.
+
+circleci/delete-me (Not published)
+circleci/delete-me-too (Not published)
+circleci/gradle (0.0.1)
+circleci/heroku (Not published)
+circleci/rollbar (0.0.1)
+
+In order to see more details about each orb, type: ` + "`circleci orb info orb-namespace/orb-name`" + `
+
+Search, filter, and view sources for all Orbs online at https://circleci.com/developer/orbs/
```

Comment:
This part would benefit from saying "except private orbs"

Reply:
In the new formatting function, I opted to just not prompt users (until we've reached parity) to take any action and eliminate these lines when `--private` is provided. Any thoughts?

Reply:
Make sense, thanks for updating!